<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1205</width>
    <height>635</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1205</width>
    <height>635</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1205</width>
    <height>635</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Reiseb√ºro Upandaway</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressource.qrc">
    <normaloff>:/main/res/icon.svg</normaloff>:/main/res/icon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>375</width>
      <height>575</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QGroupBox" name="groupBox_main">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>391</width>
      <height>580</height>
     </rect>
    </property>
    <property name="title">
     <string>Buchungsdaten</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_bookingnumber">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Buchungsnummer</string>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="calendarWidget_from">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>240</width>
       <height>165</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
     <property name="selectionMode">
      <enum>QCalendarWidget::NoSelection</enum>
     </property>
     <property name="navigationBarVisible">
      <bool>true</bool>
     </property>
     <property name="dateEditEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCalendarWidget" name="calendarWidget_to">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>245</y>
       <width>240</width>
       <height>165</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>6</pointsize>
      </font>
     </property>
     <property name="selectionMode">
      <enum>QCalendarWidget::NoSelection</enum>
     </property>
     <property name="dateEditEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>von</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>245</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>bis</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>430</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Preis</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>460</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Reise</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_customer">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>490</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>490</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Kunde</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="doubleSpinBox_price">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>430</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_travel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>460</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_details">
    <property name="geometry">
     <rect>
      <x>805</x>
      <y>10</y>
      <width>391</width>
      <height>580</height>
     </rect>
    </property>
    <property name="title">
     <string>Details</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_details_1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_details_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Text</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_details_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_details_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Text</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_details_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_details_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Text</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_details_4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>120</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_details_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Text</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>560</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Details schlie√üen</string>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1205</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFunktionen">
    <property name="title">
     <string>Funktionen</string>
    </property>
    <addaction name="actionDatei_einlesen"/>
    <addaction name="actionBuchungen_anzeigen"/>
    <addaction name="actionBuchungsliste_leeren"/>
    <addaction name="actionBuchungen_pr_fen"/>
    <addaction name="actionKundensuche"/>
    <addaction name="separator"/>
    <addaction name="actionProgramm_beenden"/>
   </widget>
   <addaction name="menuFunktionen"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDatei_einlesen">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/menu/res/open.svg</normaloff>:/menu/res/open.svg</iconset>
   </property>
   <property name="text">
    <string>Datei einlesen</string>
   </property>
  </action>
  <action name="actionBuchungen_anzeigen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/menu/res/show.svg</normaloff>:/menu/res/show.svg</iconset>
   </property>
   <property name="text">
    <string>Buchungsliste anzeigen</string>
   </property>
  </action>
  <action name="actionProgramm_beenden">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/menu/res/exit.svg</normaloff>:/menu/res/exit.svg</iconset>
   </property>
   <property name="text">
    <string>Programm beenden</string>
   </property>
  </action>
  <action name="actionBuchungsliste_leeren">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/menu/res/clear.svg</normaloff>:/menu/res/clear.svg</iconset>
   </property>
   <property name="text">
    <string>Buchungsliste leeren</string>
   </property>
  </action>
  <action name="actionKundensuche">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/menu/res/search.svg</normaloff>:/menu/res/search.svg</iconset>
   </property>
   <property name="text">
    <string>Kundensuche</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionBuchungen_pr_fen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/menu/res/audit.svg</normaloff>:/menu/res/audit.svg</iconset>
   </property>
   <property name="text">
    <string>Buchungen pr√ºfen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
